charts/huawei-csi-plugin/templates/daemonset-node.yaml

```diff
@@ -0,0 +1,121 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "huawei-csi-plugin.fullname" . }}
+  labels:
+    {{- include "huawei-csi-plugin.labels" . | nindent 4 }}
+    app.kubernetes.io/component: "csi-node"
+spec:
+  selector:
+    matchLabels:
+      {{- include "huawei-csi-plugin.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: "csi-node"
+  template:
+    metadata:
+       {{- with .Values.node.podAnnotations }}
+       annotations:
+         {{- toYaml . | nindent 8 }}
+       {{- end }}
+      labels:
+        {{- include "huawei-csi-plugin.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: "csi-node"
+    spec:
+      serviceAccountName: {{ .Values.serviceAccount.nodeName }}
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml | nindent 8 }}
+      {{- end }}
+      hostNetwork: true
+      containers:
+        - name: csi-node-driver-registrar
+          image: "{{ .Values.csi_node_driver_registrar.image.repository }}:{{ .Values.csi_node_driver_registrar.image.tag }}"
+          imagePullPolicy: {{ .Values.csi_node_driver_registrar.image.pullPolicy }}
+          args:
+            - "--csi-address=/csi/csi.sock"
+            - "--kubelet-registration-path=/var/lib/kubelet/plugins/csi.huawei.com/csi.sock"
+          resources:
+            {{- toYaml .Values.csi_node_driver_registrar.resources | nindent 12 }}
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /csi
+            - name: registration-dir
+              mountPath: /registration
+
+        - name: huawei-csi-driver
+          image: "{{ .Values.csi_driver.image.repository }}:{{ .Values.csi_driver.image.tag | default .Chart.AppVersion }}"
```

Comment:
it's nice to use a default, but if `.Chart.AppVersion` is `2.2.RC3` this won't work. The `.Chart.AppVersion` should be `v2.2.RC3`

Reply:
done :white_check_mark: